src/js/debug.js

```diff
@@ -882,8 +884,8 @@ function update() {
             "debug[7]": "pidA",
         };
 
-        DEBUG.enableFields.splice(DEBUG.enableFields.indexOf("Gyro") + 1, 0, "Attitude");
-        DEBUG.enableFields.push("Servo");
+        addArrayElementAfter(DEBUG.enableFields, "Attitude", "Gyro");
```

Comment:
`Gyro` exists and we add `Attitude` here correct?
Future developers might read it and think reverse. "add array element after attitude, adding gyro" which is wrong.

Reply:
@nerdCopter Thanks. Have changed the parameter order to be the same